assigner.py

```diff
@@ -31,11 +31,13 @@ def main():
     # Logging
     logger = logging.getLogger(__name__)
     formatstr="[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s"
-    logging.basicConfig(format=formatstr, datefmt="%H:%M:%S", stream=sys.stderr)
+    loghandlers = [logging.handlers.SysLogHandler(address='/dev/log')]
     if args.debug:
         logger.setLevel(logging.DEBUG)
+        loghandlers.append(logging.StreamHandler(stream=sys.stderr))
     else:
         logger.setLevel(logging.INFO)
+    logging.basicConfig(format=formatstr, datefmt="%H:%M:%S", handlers=loghandlers)
```

Comment:
Is the indentation off here or is git diff playing mind games?

Reply:
indentation is correct, but so is code
i could add a CRLF if thats clearer